### PR TITLE
'yum-config-manager' change flag to '--enable'

### PR DESCRIPTION
**Pull request by alexvdev**
--set-enabled and --set-disabled changed to actual --enable and --disable. (--set-enabled may come from Fedora).

### CODE REVIEW

File: engine/installation/linux/centos.md [hunk 1]
```diff
@@ -79,16 +79,16 @@ Docker from the repository.
     > which will almost certainly be an unstable one.
 
     ```bash
-    $ sudo yum-config-manager --set-enabled docker-testing
+    $ sudo yum-config-manager --enable docker-testing
     ```
```
Comment (alexvdev) [inline engine/installation/linux/centos.md]: This is right syntax on CentOS. '--set-enabled' does not work ('no such option).

File: engine/installation/linux/centos.md [hunk 2]
```diff
@@ -79,16 +79,16 @@ Docker from the repository.
...
-    command with the `--set-disabled` flag. To re-enable it, use the
-    `--set-enabled` flag. The following command disables the `testing`
+    command with the `--disable` flag. To re-enable it, use the
+    `--enable` flag. The following command disables the `testing`
     repository.
 
     ```bash
-    $ sudo yum-config-manager --set-disabled docker-testing
+    $ sudo yum-config-manager --disable docker-testing
     ```
```
Comment (alexvdev) [inline engine/installation/linux/centos.md]: This is right syntax on CentOS. '--set-disabled' does not work ('no such option').

**Comment by mstanleyjones** [comment]
Trying to build again: https://jenkins.dockerproject.org/job/docker/job/docker-github-io.b3m3d0/job/PR-1309/2/

**Comment by mstanleyjones** [comment]
I rebased your patch on `master`. Let's see if the tests pass this time.

**Pull request merged**